[IMP] l10n_ar_account_tax_settlement: ARBA archivos txt para IIBB desde 01/12/2025

pablohmontenegro opened the pull request:
Desarrollo de nuevos archivos txt para la presentación de declaraciones juradas de impuestos provinciales de ARBA.

1) 'TXT Perc/Ret IIBB aplicadas ARBA desde 01/12/2025: Percepciones ( excepto actividad 29, 7 quincenal, 7 y 17 de Bancos)'. (iibb_aplicado_arba_desde_01122025)
2) 'TXT Perc/Ret IIBB aplicadas ARBA desde 01/12/2025: Percepciones Act. 7 método Percibido (quincenal)'. (iibb_aplicado_arba_act_7_desde_01122025).
La especificación de los 2 archivos txt se puede ver en readme o en el docstring de cada uno de los métodos que los generan.
Tarea: 57326

roboadhoc commented:
[![Pull request status dashboard](https://mergebot.adhoc.ar/ingadhoc/odoo-argentina-ee/pull/789.png)](https://mergebot.adhoc.ar/ingadhoc/odoo-argentina-ee/pull/789)

pablohmontenegro committed:
[IMP] l10n_ar_account_tax_settlement: ARBA archivos txt para IIBB desde 01/12/2025
Desarrollo de nuevos archivos txt para la presentación de declaraciones juradas de impuestos provinciales de ARBA.

1) 'TXT Perc/Ret IIBB aplicadas ARBA desde 01/12/2025: Percepciones ( excepto actividad 29, 7 quincenal, 7 y 17 de Bancos)'. (iibb_aplicado_arba_desde_01122025)
2) 'TXT Perc/Ret IIBB aplicadas ARBA desde 01/12/2025: Percepciones Act. 7 método Percibido (quincenal)'. (iibb_aplicado_arba_act_7_desde_01122025).
La especificación de los 2 archivos txt se puede ver en readme o en el docstring de cada uno de los métodos que los generan.
Tarea: 57326
